cmd/parse_hcl.go

```diff
@@ -1,8 +1,8 @@
 package cmd
 
 import (
+	"github.com/gruntwork-io/go-commons/errors"
 	"github.com/gruntwork-io/terragrunt/config"
-	"github.com/gruntwork-io/terragrunt/errors"
```

Comment:
Terragrunt exported that package to the common repository in v0.52.4, I believe.